### ISSUE TITLE
lint: check global expressions with unexported-call checker

### ISSUE THREAD

**Issue by Quasilyte** [PieselBois/kfulint#103]
In it's current implementation, `unexported-call` checker only visits function bodies.
It should find invocation of "private" methods in global scope as well:
```go
package main

type foo struct{}

func (foo) unexported() int { return 1 }

var globalFoo foo // Our global object

// This line should trigger a warning:
var secretValue = foo.unexported()

func main() {}
```

In other words, it should also handle `ValueSpec` in addition to function decls.

Any unexported methods call at top level is a warning candidate, there is no "permitted" receiver there.
